lib/Wikia/src/Service/User/Attributes/AttributesModule.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Wikia\Service\User\Attributes;
+
+use Wikia\DependencyInjection\InjectorBuilder;
+use Wikia\DependencyInjection\Module;
+use Wikia\Persistence\User\Attributes\AttributePersistence;
+use Wikia\Persistence\User\Attributes\AttributePersistenceSwagger;
+
+class AttributesModule implements Module {
+	public function configure( InjectorBuilder $builder ) {
+		$builder
+			->bind( AttributeService::class )->toClass( AttributeKeyValueService::class )
+			->bind( AttributePersistence::class )->toClass( AttributePersistenceSwagger::class )
+			->bind( UserAttributes::DEFAULT_ATTRIBUTES )->to( function() {
```

Comment:
Yea, exactly. The DI library looks at that annotation to determine what to inject. It sees that we want to inject `UserAttributes::DEFAULT_ATTRIBUTES` which is bound to this function, so it injects the result of this function.